avocado/core/loader.py

```diff
@@ -36,12 +37,14 @@
 from .settings import settings
 from .output import LOG_UI
 
-#: Show default tests (for execution)
-DEFAULT = False
-#: Available tests (for listing purposes)
-AVAILABLE = None
-#: All tests (including broken ones)
-ALL = True
+
+class WhichTests(Enum):
```

Comment:
I'd suggest the use of `unique` and `auto` (or `object()`) for values. Perhaps even `IntEnum`.

Reply:
Also the name sounds a bit strange. It's fine for variable because it serves as a question, but for type it looks odd (to my non-native eyes). How about simply `Mode` so it becomes `loader.Mode.DEFAULT`? Or `Discover` (`loader.Discover.AVAILABLE`)?